modules/javafx.graphics/src/main/java/javafx/scene/SpotLight.java

```diff
@@ -39,18 +39,33 @@
 import javafx.scene.paint.PhongMaterial;
 
 /**
- * A {@code PointLight} that radiates light in a cone in a specific direction. The direction of the {@code SpotLight} is
- * defined by the {@link #directionProperty() direction} property.
+ * A {@code SpotLight} is a {@code PointLight} that radiates light in a cone in a specific direction.
+ * The direction is defined by the {@link #directionProperty() direction} vector property of the light. The direction
+ * can be rotated by setting a rotation transform on the {@code SpotLight}. For example, if the direction vector is
+ * {@code (1, 1, 1)} and the light is not rotated, it will point in the {@code (1, 1, 1)} direction, and if the light is
+ * rotated 90 degrees on the y axis, it will point in the {@code (1, 1, -1)} direction.
  * <p>
- * The light cone is defined by 3 factors: an {@link #innerAngleProperty() inner angle}, an {@link #outerAngleProperty()
- * outer angle}, and a {@link #falloffProperty() falloff} factor. For a point whose angle to the light is {@code a}, if
- * {@code a < innerAngle} then that point receives maximum illumination, if {@code a > outerAngle} then that point
- * receives no illumination, and if {@code innerAngle <= a <= outerAngle} then the illumination is determined by the
- * formula
- * <pre>I = pow((cos(a) - cos(outer)) / (cos(inner) - cos(outer)), falloff)</pre>
- * which represents a drop in illumination from the inner angle to the outer angle. {@code falloff} determines the
- * behavior of the drop. The valid ranges for these properties are {@code 0 <= innerAngle <= outerAngle <= 180} and
- * {@code falloff >= 0}; values outside either of these ranges can produce unexpected results.
+ * In addition to the factors that control the light intensity of a {@code PointLight}, a {@code SpotLight} has a
+ * light-cone attenuation factor, {@code spot}, that is determined by 3 properties:
+ * <ul>
+ * <li> {@link #innerAngleProperty() inner angle}: the angle of the inner cone (see image below)
+ * <li> {@link #outerAngleProperty() outer angle}: the angle of the outer cone (see image below)
```

Comment:
I recommend changing these to `innerAngle` and `outerAngle` (one word, camel case) to match the names of the properties.